drivers/overlay/ov_network.go

```diff
@@ -173,9 +256,11 @@ func (n *network) initSandbox() error {
 	})
 
 	go n.watchMiss(nlSock)
-	return n.initVxlan()
+	return nil
+	//return n.initVxlan()
```

Comment:
Even if there is only one caller sometimes we have a separate function so that the caller function doesn't get too long or get into too many indentation levels. So it is perfectly fine to remove these commented lines